src/mockServiceWorker.js

```diff
@@ -11,16 +11,16 @@
 const INTEGRITY_CHECKSUM = '<INTEGRITY_CHECKSUM>'
 const activeClientIds = new Set()
 
-self.addEventListener('install', function () {
-  self.skipWaiting()
+self.addEventListener('install', async function () {
+  await self.skipWaiting()
```

Comment:
Take a look at the [.skipWaiting()](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerGlobalScope/skipWaiting) page on MDN. Although some service worker scope methods return promises, those are implicitly tied to the worker internals and you don't have to await them. In fact, some event listeners, like "fetch", must be entirely synchronous otherwise they will throw. This means they can still execute async things sequentially but rid you of the need to await them explicitly.